### PR TITLE
add pymudpdf

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/PyMuPdf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - swig
+    - mupdf
```
Comment (scopatz) [inline recipes/PyMuPdf/meta.yaml:21]: mupdf should be in host

Comment (scopatz) [reply recipes/PyMuPdf/meta.yaml:21]: swig maybe should as well.

File: recipes/PyMuPdf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{% set name = "PyMuPdf" %}
+{% set version = "1.17.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/e8/a8/c5774138aa34f1fe24d9efeb1506f148bb55c52583bf44b0f9d48cbcd59e/PyMuPDF-1.17.3.tar.gz
```
Comment (scopatz) [inline recipes/PyMuPdf/meta.yaml:10]: Is there a reason that `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz` doesn't work?

Comment (hadim) [reply recipes/PyMuPdf/meta.yaml:10]: You can try it yourself, it does not work. Probably a lowercase/uppercase issue.

**Comment by hadim** [comment]
I don't think the mupdf package provides include files. So I'll probably close this PR for now:

```
Failed to build PyMuPDF
Installing collected packages: PyMuPDF
  Created temporary directory: /tmp/pip-record-9k9x90zb
    Running command /home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-pbqmf_1c/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-pbqmf_1c/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-9k9x90zb/install-record.txt --single-version-externally-managed --compile --install-headers /home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/include/python3.7m/PyMuPDF
    Running setup.py install for PyMuPDF: started
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.7
    creating build/lib.linux-x86_64-3.7/fitz
    copying fitz/__init__.py -> build/lib.linux-x86_64-3.7/fitz
    copying fitz/fitz.py -> build/lib.linux-x86_64-3.7/fitz
    copying fitz/utils.py -> build/lib.linux-x86_64-3.7/fitz
    copying fitz/__main__.py -> build/lib.linux-x86_64-3.7/fitz
    running build_ext
    building 'fitz._fitz' extension
    creating build/temp.linux-x86_64-3.7
    creating build/temp.linux-x86_64-3.7/fitz
    /home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_build_env/bin/x86_64-conda_cos6-linux-gnu-cc -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O2 -Wall -Wstrict-prototypes -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/include -fdebug-prefix-map=/home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/work=/usr/local/src/conda/pymupdf-1.17.3 -fdebug-prefix-map=/home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla=/usr/local/src/conda-prefix -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -isystem /home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/include -fPIC -I/usr/include/mupdf -I/usr/local/include/mupdf -I/home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/include/python3.7m -c fitz/fitz_wrap.c -o build/temp.linux-x86_64-3.7/fitz/fitz_wrap.o
    fitz/fitz_wrap.c:2733:10: fatal error: fitz.h: No such file or directory
     #include <fitz.h>
              ^~~~~~~~
    compilation terminated.
    error: command '/home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_build_env/bin/x86_64-conda_cos6-linux-gnu-cc' failed with exit status 1
    Running setup.py install for PyMuPDF: finished with status 'error'
ERROR: Command errored out with exit status 1: /home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-pbqmf_1c/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-pbqmf_1c/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-9k9x90zb/install-record.txt --single-version-externally-managed --compile --install-headers /home/conda/staged-recipes/build_artifacts/pymupdf_1594944219485/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/include/python3.7m/PyMuPDF Check the logs for full command output.
Exception information:
Traceback (most recent call last):
  File "$PREFIX/lib/python3.7/site-packages/pip/_internal/req/req_install.py", line 841, in install
    req_description=str(self.req),
  File "$PREFIX/lib/python3.7/site-packages/pip/_internal/operations/install/legacy.py", line 86, in install
    raise LegacyInstallFailure
pip._internal.operations.install.legacy.LegacyInstallFailure

```

**Comment by scopatz** [comment]
I think the include files are there but the build system for pymudf needs to be patched somehow.  I built the mupdf package and gave up on building pymupdf

**Pull request closed without merging**